monitoring/shutdown-javaagent/src/main/java/com/google/cloud/runtimes/monitoring/shutdown/config/SysEnvironment.java

```diff
@@ -0,0 +1,11 @@
+package com.google.cloud.runtimes.monitoring.shutdown.config;
+
+public class SysEnvironment {
```

Comment:
javadoc.

I guess this class is mostly so you can mock out the environment variables in unit tests?